components/member-profile/index.js

```diff
@@ -1,29 +1,25 @@
-import { getDataURL, getImgURL } from 'helper-functions/urls';
-
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from './member-profile.module.scss';
-import useFetch from 'custom-hooks/useFetch';
 
 const Profile = (props) => {
-  const imgLink = getImgURL(props.id);
-  const { data } = useFetch(getDataURL(props.id));
+  const { membersData, imageLink } = props;
+  const memberName = `${membersData.first_name || '--'} ${membersData.last_name || '--'}`;
```

Comment:
Great if you refer my old comment. 

If the memberData is missing either `last_name` or `first_name` she/he will see `--`, which is not good.